features/testbot/distance_matrix.feature

```diff
@@ -17,9 +17,9 @@ Feature: Basic Distance Matrix
             | ab    |
 
         When I request a travel distance matrix I should get
-            |   | a   | b   |
-            | a | 0   | 100 |
-            | b | 100 | 0   |
+            |   | a      | b      |
+            | a | 0      | 100+-1 |
```

Comment:
Elsewhere (like in the "When I route I should get " tests) the syntax used is "100m +-3", rather than "100+-3". For consistency it might be good to follow this syntax.